--- minify-3.0/lib/Minify/YUICompressor.php~	2016-09-13 14:48:49.356998362 +0300
+++ minify-3.0/lib/Minify/YUICompressor.php	2016-09-13 14:49:07.236739393 +0300
@@ -37,7 +37,7 @@
      *
      * @var string
      */
-    public static $jarFile = null;
+    public static $jarFile = '/usr/share/java/yuicompressor.jar';
 
     /**
      * Writable temp directory. This must be set before calling minifyJs()
